examples/shell/shell_common/cmd_device.cpp

```diff
@@ -962,8 +962,10 @@ void cmd_device_init()
 
 #if CHIP_DEVICE_CONFIG_ENABLE_THREAD
     streamer_printf(sout, "Init Thread stack\r\n");
-    error = ThreadStackMgr().InitThreadStack();
-    VerifyOrExit(error == CHIP_NO_ERROR, streamer_printf(sout, "ThreadStackMgr().InitThreadStack() failed\r\n"));
+    if (ThreadStackMgr().InitThreadStack() != CHIP_NO_ERROR)
+    {
+        streamer_printf(sout, "ThreadStackMgr().InitThreadStack() failed\r\n");
```

Comment:
nit (not your code, but noticing now): is the "\r" needed given that in other places I saw only "\n" used?

Reply:
'\r' is not needed since we don't plan to run this shell app on Windows